client/my-sites/plans/current-plan/index.jsx

```diff
@@ -220,7 +221,8 @@ class CurrentPlan extends Component {
 		const currentPlanSlug = selectedSite?.plan?.product_slug ?? '';
 		const isEcommerceTrial = currentPlanSlug === PLAN_ECOMMERCE_TRIAL_MONTHLY;
 		const isBusinessTrial = currentPlanSlug === PLAN_MIGRATION_TRIAL_MONTHLY;
```

Comment:
It doesn't look like `isBusinessTrial` constant doesn't look like it's used anywhere else. But I think both the hosting and migration trial should count as "business trial". That was sort of the original conception.

```suggestion
		const isBusinessTrial = currentPlanSlug === PLAN_MIGRATION_TRIAL_MONTHLY || currentPlanSlug === PLAN_HOSTING_TRIAL_MONTHLY`
```